ui/page/info/info_page.go

```diff
@@ -86,58 +115,185 @@ func NewInfoPage(l *load.Load) *WalletInfo {
+
+	pg.loadTransactions()
+
+	if pg.wallet.GetAssetType() == libutils.DCRWalletAsset {
+		pg.loadStakes()
+
+		if pg.wallet.(*dcr.Asset).IsAccountMixerActive() {
+			pg.listenForMixerNotifications()
+			pg.reloadMixerBalances()
+		}
+	}
 }
 
 // Layout draws the page UI components into the provided layout context
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
 // Layout lays out the widgets for the main wallets pg.
-func (pg *WalletInfo) Layout(gtx layout.Context) layout.Dimensions {
+func (pg *WalletInfo) Layout(gtx C) D {
 	return pg.Theme.List(pg.container).Layout(gtx, 1, func(gtx C, i int) D {
 		return layout.Inset{Right: values.MarginPadding2}.Layout(gtx, func(gtx C) D {
-			return pg.Theme.Card().Layout(gtx, func(gtx C) D {
-				return layout.UniformInset(values.MarginPadding20).Layout(gtx, func(gtx C) D {
-					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-						layout.Rigid(func(gtx C) D {
-							return layout.Inset{
-								Right: values.MarginPadding10,
-								Left:  values.MarginPadding10,
-							}.Layout(gtx, func(gtx C) D {
-								txt := pg.Theme.Body1(pg.WL.SelectedWallet.Wallet.GetWalletName())
-								txt.Font.Weight = font.SemiBold
-								return txt.Layout(gtx)
-							})
-						}),
-						layout.Rigid(func(gtx C) D {
-							if len(pg.WL.SelectedWallet.Wallet.GetEncryptedSeed()) > 0 {
-								return layout.Inset{
-									Top: values.MarginPadding16,
-								}.Layout(gtx, func(gtx C) D {
-									return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
-										layout.Rigid(pg.Theme.Icons.RedAlert.Layout24dp),
-										layout.Rigid(func(gtx C) D {
-											return layout.Inset{
-												Left:  values.MarginPadding9,
-												Right: values.MarginPadding16,
-											}.Layout(gtx, pg.Theme.Body2(values.String(values.StrBackupWarning)).Layout)
-										}),
-										layout.Rigid(pg.toBackup.Layout),
-									)
-								})
-							}
+			items := []layout.FlexChild{layout.Rigid(pg.walletInfoLayout)}
+
+			if pg.wallet.GetAssetType() == libutils.DCRWalletAsset && pg.wallet.(*dcr.Asset).IsAccountMixerActive() {
+				items = append(items, layout.Rigid(pg.mixerLayout))
+			}
+
+			if len(pg.transactions) > 0 {
+				items = append(items, layout.Rigid(pg.recentTransactionLayout))
+			}
+
+			if len(pg.stakes) > 0 {
+				items = append(items, layout.Rigid(pg.recentStakeLayout))
+			}
+
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx, items...)
+		})
+	})
+}
+
+func (pg *WalletInfo) walletInfoLayout(gtx C) D {
+	return pg.pageContentWrapper(gtx, "", func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(pg.walletNameAndBackupInfo),
+			layout.Rigid(pg.syncStatusSection),
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+					layout.Flexed(1, func(gtx C) D {
+						return layout.E.Layout(gtx, pg.toBackup.Layout)
+					}),
+				)
```

Comment:
don't need  wrap backup layout button to flex

Reply:
The end game is to overwrited the default layout alignment and force the button the align to the east, as the default layout alignment is to the west.